### ISSUE TITLE
Documentation suggestion related to client.DB

### ISSUE THREAD

**Issue by fabiante** [go-kivik/kivik#575]
Hi, I'd like to make a suggestion for the documentation, specifically `kivik.DB`. 

TLDR: Is it safe / cheap to create lots of `kivik.DB` instances from `client.DB`? Should that be documented?

I want to use CouchDB in a multi tenant capable web service. My current implementation follows the idea of instantiating one `Service` for each incoming request. As care more about simplicity than highes efficiency I prefer this solution because it let's me implement a kind of project structure that other people can also under stand.

```go
type Service struct {
    db *kivik.DB
}

func NewService(db *kivik.DB) *Service {
	return &Service {db: db}
}

func (s *Service) DoSomething() {
    // .... my application logic
}
```

Now, what has this to do with kivki? For my implementation to work, I want to always create `Service` with the `*kivik.DB` that matches for the request tenant. When building a prototype, I was unsure if it is fine to call [`client.DB`](https://pkg.go.dev/github.com/go-kivik/kivik/v3#Client.DB) lots of times, creating one `DB` instance per request. Sure, I already see that this is a solution with the potential of optimization. But still, I want to know how the library would potentially behave.

As far as I understand the source code of kivik and the couchdb package, calling `client.DB` will create a new a new `DB` value via the used driver. Generally it seems to me that it should be fine to instantiate lots of these instances as there is no special concurrency.

So I guess it is both safe and "cheap" to instantiate multiple DB instances? If so, I though it might be helpful for others to add this as documentation to the `client.DB` method. I would be glad to help.

**Comment by flimzy** [comment]
I would agree that it should be safe and cheap to keep multiple `DB` instances around.  The interesting logic/concurrency happens in the `couchdb.Client` object, which is just pointed to by the `DB` object. And in reality, the interesting stuff even there is all handled by `net/http.Client`.

It should also be cheap to (re-)create `DB` objects as needed, since all that really does under the hood is correlate a dbname with an existing `couchdb.Client` object.

**Comment by FabianTe** [comment]
Hi, thanks for your answer. Happy that you confirm my assumptions. IMO it is always helpful to adress concurrency / thread safety on libraries so that consumers know how a library is expected to be handled. In my personal case I am happy that kivik is easy to deal with :) 

I can offer you to add this information to the code documentation and maybe the wiki (I'd create a seperate branch with a markdown file so you can review that and manually use the changes).
